packages/core/src/App/Components/Elements/NotificationMessage/timeout-border-bottom.jsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Localize } from '@deriv/translations';
+
+const LinearProgress = ({ progress }) => (
+    <div className={classNames('dc-linear-progress')}>
```

Comment:
can this be in Components package instead?

Reply:
Will do. Thanks for the suggestion.